tests/extension/provider/launcher.test.ts

```diff
@@ -0,0 +1,40 @@
+import { vi, describe, it, expect } from 'vitest';
```

Comment:
Nit: Looks like this file need a eslint fix-all run. Based on the current rules I won't pass linting.

Reply:
Fixed